client/blocks/product-purchase-features/product-purchase-features-list/index.jsx

```diff
@@ -43,14 +43,16 @@ class ProductPurchaseFeaturesList extends Component {
 	static propTypes = {
 		plan: PropTypes
 			.oneOf( [ PLAN_FREE,
-				PLAN_PERSONAL,
 				PLAN_PREMIUM,
 				PLAN_BUSINESS,
 				PLAN_JETPACK_FREE,
 				PLAN_JETPACK_BUSINESS,
 				PLAN_JETPACK_BUSINESS_MONTHLY,
 				PLAN_JETPACK_PREMIUM,
-				PLAN_JETPACK_PREMIUM_MONTHLY ] )
+				PLAN_JETPACK_PREMIUM_MONTHLY,
+				PLAN_PERSONAL,
```

Comment:
Did you mean `PLAN_JETPACK_PERSONAL` here? I'm seeing the same error @tyxla mentioned with a yearly personal plan.